app/routes/contents.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Content = require('../models/Content');
+const Router = require('express').Router;
+
+const router = Router();
+
+router.post('/', (req, res) => {
+  if(!req.body.productId){
+    res.status(404).send();
```

Comment:
We need to return here so we don't attempt to "respond twice". (2nd response will throw an error trying to respond again)